### PR TITLE
feat: update files and folders to follow rebranding

### PR DESCRIPTION
**Pull request by converge**
this PR:

- update `~/.flare` to `~/.kubefirst`
- update `~/.kubefirst` folder to `~/.k1srt`

Signed-off-by: João Vanzuita <joao@kubeshop.io>

### CODE REVIEW

File: cmd/clean.go [hunk 1]
```diff
@@ -26,15 +26,25 @@ to quickly create a Cobra application.`,
...
+		}
+
+		err = os.Remove(config.KubefirstConfigFilePath)
+		if err != nil {
+			log.Panicf("unable to delete %q file, error is: ", err)
+		}
 		log.Println("removed $HOME/.kubefirst and $HOME/.flare")
-		if err := os.Mkdir(fmt.Sprintf("%s/.kubefirst", config.HomePath), os.ModePerm); err != nil {
-			log.Panicf("error: could not create directory $HOME/.kubefirst - it must exist to continue %s", err)
+
```
Comment (6za) [inline cmd/clean.go:39]: does this log need to be updated?

File: cmd/clean.go [hunk 2]
```diff
@@ -26,15 +26,25 @@ to quickly create a Cobra application.`,
 
 		log.Println("removing $HOME/.kubefirst and $HOME/.flare")
 		// todo ask for user input to verify?
```
Comment (6za) [inline cmd/clean.go:28]: Maybe should we use the configs? instead of the name in the log string...

**Comment by 6za** [comment]
Minor comments on clean step referring to what I guess is the old naming.

File: cmd/clean.go (hunk 1 above)
Comment (converge) [reply cmd/clean.go]: nice catch! It was updated

File: cmd/clean.go (hunk 2 above)
Comment (converge) [reply cmd/clean.go]: that's true, It was updated

**Pull request merged**